middleware.go

```diff
@@ -196,11 +195,11 @@ func Session(cfg Options) func(next http.Handler) http.Handler {
 					if adminClaim {
 						sessionType = proto.SessionType_Admin
 					}
+				}
 
-					if projectClaim > 0 {
-						ctx = WithProjectID(ctx, uint64(projectClaim))
-						sessionType = proto.SessionType_Project
-					}
+				if projectClaim > 0 {
+					ctx = WithProjectID(ctx, uint64(projectClaim))
+					sessionType = max(sessionType, proto.SessionType_Project)
```

Comment:
How do we distinguish between AccessKey and Project sessions?

Should we have a "session" claim?

Reply:
The sessionType is accessKey vs project